libs/polysemy-wire-zoo/src/Wire/Sem/Now/Input.hs

```diff
@@ -18,22 +18,17 @@
 -- You should have received a copy of the GNU Affero General Public License along
 -- with this program. If not, see <https://www.gnu.org/licenses/>.
 
-module Wire.Sem.Now.Input
-  ( nowToInput,
-  )
-where
+module Wire.Sem.Now.Input (nowToInput) where
 
 import Data.Time (UTCTime)
 import Imports
 import Polysemy
 import Polysemy.Input
-import Wire.Sem.FromUTC (FromUTC (..))
 import Wire.Sem.Now
 
 nowToInput ::
-  FromUTC t =>
```

Comment:
I think this was the only place we ever used this class and I think this makes it redundant.

Reply:
yeah I was wrong; there's `boolTTL` :eyes: